src/client/Microsoft.Identity.Client/Http/Retry/DefaultRetryPolicy.cs

```diff
@@ -52,6 +46,11 @@ public DefaultRetryPolicy(RequestType requestType)
             }
         }
 
+        internal virtual Task DelayAsync(int milliseconds)
```

Comment:
Is this method still needed? Does it need to be virtual?

Reply:
Yes, the TestDefaultRetryPolicy overrides that method to resolve the Task instantly instead of pausing for a delay